qucs-doc/report/EKV26/Makefile.am

```diff
@@ -29,6 +29,8 @@ EXTRA_DIST = $(SOURCES)
 SOURCES = EKV26.tex content.tex
 TARGETS = EKV26.pdf
 
+export TEXINPUTS=$(srcdir):$(srcdir)/..:
```

Comment:
I did differently because the `Makefile.am` is less cluttered in this way; no need to have a variable repeated for every invocation of `pdflatex` (and for the `.tex.dvi` rules, even if they are not used at the moment, I think).
Why exporting `TEXINPUTS` here is bad? To me it seems almost the same as you did, every recipe line invoking `LaTeX` will be executed in its own shell,and the environment will have `TEXINPUTS` set automatically instead of having to specify it explicitly.

Your commit is not in this PR because I did not actually start from it (to be pedantic, I should have written "...expanded from the idea in #524"). I can certainly add it.